samples/login-sample/src/main/java/com/uber/sdk/android/samples/LoginSampleActivity.java

```diff
@@ -127,14 +140,35 @@ protected void onCreate(Bundle savedInstanceState) {
                 new SampleLoginCallback(),
                 configuration,
                 CUSTOM_BUTTON_REQUEST_CODE);
-
         customButton = (Button) findViewById(R.id.custom_uber_button);
         customButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 loginManager.login(LoginSampleActivity.this);
             }
         });
+        appLinkButton = findViewById(R.id.applink_uber_button);
+        appLinkButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(Intent.ACTION_VIEW);
```

Comment:
This seems like more boilerplate than we may want a dev to write. See loginManager.login as an example where we abstracted startActivityForResult handling. It also provides a proxy activity to catch and return callbacks. We probably want to do something similar.

Reply:
This is intentionally done here.
With applink support devs may not use the SDK to launch single sign on with Uber app. So, this piece is added to guide the devs how they can assemble the applink and use startActivityForResult to launch it